client/my-sites/stats/post-trends/day.jsx

```diff
@@ -103,10 +96,10 @@ module.exports = React.createClass( {
 				<Popover context={ this.refs && this.refs.day }
 					isVisible={ this.state.showPopover }
 					position="top"
-					onClose={ noop }
 					className="chart__tooltip is-streak"
+					onClose={ noop }
```

Comment:
Indeed I did try to remove this, and of course eslint complains about the `() => {}` here.  I agree we should make this an optional prop with a default of a `noop` in the base component.  Are you okay with doing this in a follow-up?